db/schema.rb

```diff
@@ -67,6 +67,16 @@
     t.index ["item_id"], name: "index_reviews_on_item_id"
   end
 
+  create_table "users", force: :cascade do |t|
+    t.string "name"
+    t.string "address"
+    t.string "city"
+    t.string "state"
+    t.string "zip"
+    t.string "email"
+    t.string "password_digest"
+  end
```

Comment:
I think it is worth getting more clarification about the standard for where User data lives & how it is typically structured.

Reply:
agreed.